asf_core_data/pipeline/data_joining/merge_proc_datasets.py

```diff
@@ -222,16 +226,36 @@ def merging_pipeline(
         epc_mcs_complete, postcode_var_name="POSTCODE", white_space="remove"
     )
 
-    if save:
-        # Save final merged dataset
-        data_getters.save_to_s3(
-            base_config.BUCKET_NAME,
-            epc_mcs_complete,
-            base_config.EPC_MCS_MERGED_OUT_PATH,
-        )
+    today = date.today().strftime("%y%m%d")
 
-    return epc_mcs_complete
+    # Save final merged dataset
+    data_getters.save_to_s3(
+        base_config.BUCKET_NAME,
+        epc_mcs_complete,
+        base_config.EPC_MCS_MERGED_OUT_PATH.format(today),
+    )
+
+
+def create_argparser() -> ArgumentParser:
+    """
+    Creates an argument parser that can receive the following arguments:
+    - path_to_data: either local path to where data is stored or "S3"
+    """
+    parser = ArgumentParser()
+
+    parser.add_argument(
+        "--path_to_data",
+        help="Path to data",
+        default="S3",
+        type=str,
+    )
+
+    return parser
 
 
 if __name__ == "__main__":
-    merging_pipeline()
+    parser = create_argparser()
+    args = parser.parse_args()
+
+    path_to_data = args.path_to_data
+    merging_pipeline(path_to_data=path_to_data)
```

Comment:
If you want to save a line 

```suggestion
    merging_pipeline(path_to_data=args.path_to_data)
```